server/src/main/java/chat/engine/Connect.java

```diff
@@ -14,6 +14,8 @@
 
     private static final String EXIT = "exit";
 
+    private static final String PM = "'";
```

Comment:
you can move your commands constants to separate class, for storing them in one place